driver/test/statement_parameters_it.cpp

```diff
@@ -122,7 +122,7 @@ TEST_F(StatementParametersTest, BindingNullStringValueForInteger) {
 #    define SQL_C_myTCHAR SQL_C_TCHAR
 #endif
 
-    auto param = fromUTF8<SQLmyTCHAR>("Null");
+    auto param = fromUTF8<SQLmyTCHAR>("\\N");
```

Comment:
Update the PR name / desciption if you want to fix that in that PR. 

BTW - is there any tests for that functionality?

Reply:
Yes, this is a fix of those test, that fail with new server.